atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraClientRateLimitingConfig.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.cassandra;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+@JsonDeserialize(as = ImmutableCassandraClientRateLimitingConfig.class)
+@JsonSerialize(as = ImmutableCassandraClientRateLimitingConfig.class)
+@Value.Immutable
+public interface CassandraClientRateLimitingConfig {
+    @Value.Default
+    default int maxConcurrentRangeScans() {
+        return 100;
```

Comment:
very high to start with - what should I bring this down to?

Reply:
Sent some internal graphs over